lib/NativeCode/DynamicLibraryLoaderHelper/NativeRender/dllmain.cpp

```diff
@@ -1353,11 +1373,18 @@ void eos_create(EOSConfig& eosConfig)
         steamApiInterfaceVersionsAsCharArray.push_back('\0');
 
         steam_platform.SteamApiInterfaceVersionsArray = reinterpret_cast<char*>(steamApiInterfaceVersionsAsCharArray.data());
+                
+        auto size = steamApiInterfaceVersionsAsCharArray.size();
+
+        if (size > (std::numeric_limits<uint32_t>::max)()) {
```

Comment:
I should have done this when setting this up, but if we're doing this comparison; there's also a maximum size this message can be, maybe we should also check against that.

Reply:
https://dev.epicgames.com/docs/en-US/api-ref/structs/eos-integrated-platform-steam-options

> Size of the SteamApiInterfaceVersionsArray in bytes. Cannot exceed EOS_INTEGRATEDPLATFORM_STEAM_MAX_STEAMAPIINTERFACEVERSIONSARRAY_SIZE.

Reply:
Also place the open curly on a new line.

Reply:
The if statement now just compares against the aforementioned constant.